src/main/java/dbaccessor/message/MessageAccessor.java

```diff
@@ -242,4 +243,24 @@ public boolean checkIfMessageIdBelongsToChatId(long messageId, long chatId) {
  
         return resultSet;
     }
+
+    /**
+     * Returns details of Attachment corresponding to the Message with the given MessageId.
+     * Details include:
+     * <ol>
+     * <li> AttachmentId </li>
+     * <li> FileName </li>
+     * <li> FileType </li>
+     * <li> FileSize </li>
+     * </ol>
+     */
+    public Attachment getAttachmentFromMessageId(long messageId) {
+        String sqlStatment = "SELECT Attachment.AttachmentID as AttachmentID, Attachment.FileName as FileName, Attachment.FileType as FileType, Attachment.FileSize as FileSize FROM Message LEFT OUTER JOIN Attachment ON Message.AttachmentID = Attachment.AttachmentID WHERE Message.MessageID = @messageId";
+        Statement statement = Statement.newBuilder(sqlStatment).bind("messageId").to(messageId).build();
+        ImmutableList<Attachment> resultSet = ImmutableList.<Attachment>builder() 
```

Comment:
If you are just going to return the first element, there is no point in copying it to an `ImmutableList`.

Reply:
Yes, sorry :D. Made the change.